packages/plugins/src/plugins/diagnostic-check.plugin.ts

```diff
@@ -23,9 +23,8 @@ export class DiagnosticCheckPlugin extends Plugin {
     DEBUG_CALLBACK(`Plugin 'DiagnosticCheck' run on %O:`, fileName);
 
     const allFixedFiles: Set<string> = new Set();
-    let tries = diagnostics.length + 1;
```

Comment:
Tries were needed for cases when @ts-ignore didn't work.
We don't rely on it anymore and don't need this countdown anymore.